src/libraries/Microsoft.Extensions.Options/tests/Microsoft.Extensions.Options.Tests/OptionsBuilderTest.cs

```diff
@@ -665,5 +665,125 @@ public void CanValidateMixDataAnnotations()
                 "DataAnnotation validation failed for members: 'Dep1,Dep2' with the error: 'Dep1 != Dep2'.",
                 "I don't want to go to nowhere!");
         }
+
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34580", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
This issue is closed as a dupe of #34582. Can we update the link to point to the active issue?

Reply:
I'll address this in another PR there are plenty of tests currently linked with the dupe.